ui-components/contextform.md

```diff
@@ -57,7 +57,7 @@ Unlike normal context toolbar buttons, Context form buttons are not registered b
 
 The definition of a context form button is very similar to the definition of a normal toolbar button. The main difference is that the action is slightly different (as it will relate to the form), and the type is `contextformbutton` instead of `button`. The following is the full list of options:
 
-##### type: 'contextformtogglebutton';
+#### type: 'contextformtogglebutton';
```

Comment:
This is existing and maybe this has been noted before, but I think this should be `contextformbutton` and the one below should be `contextformtogglebutton` to fix the spelling mistake?

Reply:
Fixed here? https://github.com/tinymce/tinymce-docs/pull/1705/commits/0061fd092fb24de56f8a73fa3d422e86916af30e

Reply:
It is worth noting that the table for `contextformbutton` looks to be incorrect as well. I will create a Jira so this can be looked into and fixed separately.